src/java.base/windows/native/libnio/ch/Net.c

```diff
@@ -157,6 +157,7 @@ Java_sun_nio_ch_Net_socket0(JNIEnv *env, jclass cl, jboolean preferIPv6,
         SetHandleInformation((HANDLE)s, HANDLE_FLAG_INHERIT, 0);
 
         /* IPV6_V6ONLY is true by default */
+        /* attempt to disable IPV6_V6ONLY to ensure dual-socket support; ignore errors */
```

Comment:
You can the previous comment at L159.

Reply:
I adjusted the comment.

Reply:
I mean L159 can be removed, I don't think we need that now.